buildscripts/build.cmd

```diff
@@ -19,13 +19,17 @@ IF NOT EXIST %~dp0..\Settings.proj GOTO msbuild_not_configured
 REM Set Framework version based on passed in parameter
 IF "%1" == "" goto no_nothing
 
-REM nesting the netcore build in an IF body prevents reporting of the error level
+REM nesting the build command in an IF body prevents reporting of the error level
 IF /i "%1" NEQ "NETCORE" goto not_netcore
-
 PowerShell.exe -NoProfile -NonInteractive -NoLogo -ExecutionPolicy Unrestricted -Command %~dp0Build-NetCore.ps1
 EXIT /B %ERRORLEVEL%
 
 :not_netcore
+IF /i "%1" NEQ "MONO" goto not_mono
+xbuild /p:Configuration=NET45-Release /t:RunAllTests buildscripts/Build.proj
+EXIT /B %ERRORLEVEL%

```

Comment:
I know, my answer to everything is to add a new label with an explicit command, but xbuild doesn't support `/m` or `/ToolsVersion`. By the time I replaced the command and the two switches, the `build.cmd` was even messier. I could go that route if you prefer.